preprocessing/nextclade/src/loculus_preprocessing/prepro.py

```diff
@@ -727,9 +740,11 @@ def process_single(  # noqa: C901
             )
 
         submitter = unprocessed.inputMetadata["submitter"]
+        group_id = int(str(unprocessed.inputMetadata["group_id"]))
```

Comment:
```suggestion
        group_id = int(unprocessed.inputMetadata["group_id"])
```

Is there a reason for the str first? I can't think of one.